### PR TITLE
Refactor to use hydra-works characterization.

### PR DESCRIPTION
**Pull request by grosscol**
Update to hydra-works ch12n in specs.
Remove fits spec and ch12n_service spec.
  They are redundant from hydra-works.
Remove fits datastream and ch12n_service.
  They are now proivded by hydra-works.
Remove expectation of fitsdatastream from fileset.
  fitsdatastream is not longer persisted.

**Look carefully.  I do not yet trust the tests.**

### CODE REVIEW

File: spec/models/file_set_spec.rb [hunk 1]
```diff
@@ -107,10 +103,15 @@
       expect(subject).to respond_to(:filename)
       expect(subject).to respond_to(:original_checksum)
       expect(subject).to respond_to(:well_formed)
+      expect(subject).to respond_to(:page_count)
+    end
+
+    it 'has properties from characterization metadata' do
+      pending 'Resoultion of issue #226 in hydra-works'
```
Comment (jcoyne) [inline spec/models/file_set_spec.rb]: projecthydra-labs/hydra-works#226 is closed.

File: spec/models/checksum_audit_log_spec.rb [hunk 2]
```diff
@@ -3,13 +3,14 @@
...
     # stub out characterization so it does not get audited
-    allow_any_instance_of(CurationConcerns::CharacterizationService).to receive(:characterize)
+    allow_any_instance_of(Hydra::Works::CharacterizationService).to receive(:characterize)
   end
 
   let(:f) do
     file = FileSet.create do |gf|
       gf.apply_depositor_metadata('mjg36')
     end
+    # TODO: Mock addition of file to fileset.
```
Comment (jcoyne) [inline spec/models/checksum_audit_log_spec.rb]: please add a reason why we want to do this in the future to the comment.

Comment (mjgiarlo) [inline spec/models/checksum_audit_log_spec.rb]: :+1:

**Comment by mjgiarlo** [comment]
Travis is :unamused:

Pls clarify the TODO that @jcoyne mentioned, unpend the file_set spec test and get it passing, and point at hydra-derivatives instead of hydra-works for the FT extraction service.

**Comment by jcoyne** [comment]
Looks like a rebase is required too.

**Comment by mjgiarlo** [comment]
@jcoyne :shipit: ?

**Pull request merged**